docs/cudf/README.md

```diff
@@ -3,4 +3,4 @@
 This directory contains the documentation of cuDF Python.
 For more information on how to write, build, and read the documentation,
 see 
-[the developer documentation](https://github.com/rapidsai/cudf/blob/main/docs/cudf/source/developer_guide/documentation.md).
+[the developer documentation](https://github.com/rapidsai/cudf/blob/HEAD/docs/cudf/source/developer_guide/documentation.md).
```

Comment:
Was this just intended to proof against the name of the main branch changing again in the future?

Reply:
Ah I just [saw your relevant comment on the issue](https://github.com/rapidsai/cudf/issues/11348#issuecomment-1198664504). All good.